lib/assent/strategies/azure_ad.ex

```diff
@@ -7,6 +7,7 @@ defmodule Assent.Strategy.AzureAD do
   - `:client_id` - The OAuth2 client id, required
   - `:tenant_id` - The Azure tenant ID, optional, defaults to `common`
   - `:nonce` - The session based nonce, required
+  - `:response_type` - The response type to request, defaults to `id_token code`
```

Comment:
This is backwards compatable since the default is the same as what the value was before this change.